### ISSUE TITLE
Instance_Segmenter?

### ISSUE THREAD

**Issue by sayanmutd** [pjreddie/darknet#1064]
How do i use instance_segmenter? Could you specify the cfg and weight files and what is the expected output? @pjreddie  @lilohuang

**Comment by aimhabo** [comment]
The same problem, I don't know how to use the iseg layer

**Comment by sayanmutd** [comment]
I don't know which configuration and weight files to use for running the instance segmenter demo? @aimhabo @pjreddie @AlexeyAB

**Comment by aimhabo** [comment]
@sayanmutd  As far as the files are currently seen, it is not a complete process. but you

**Comment by sayanmutd** [comment]
@aimhabo  Yeah, You are right... that can be done but as it was given in the present darknet i thought some examples might be there... and without proper example you cannot guarantee about what would be expected outcome.

**Comment by aimhabo** [comment]
I can run iseg_layer now, but loss starts from 550,000, and blow-up to NaN after 500 times (64batch/32sub).
Does it ordinary?
what's the meaning of 'ids'?  Why set it to 32 in default?

**Comment by aimhabo** [comment]
now problem becomes "Delta does not change effectively, causing loss to not converge (although not divergent)"

**Comment by zouwen198317** [comment]
> I have changed the darknet use to image segmentaion, if you have interest, you can see my code: https://github.com/ArtyZe/yolo_segmentation. If you think it's useful, please give me a star :)

Hi, where r u  cfg  ? and  could you share your  cfg file?

**Comment by abhigoku10** [comment]
@zouwen198317  did u get the cfg and weight file 
@ArtyZe  can you share the cfg and weight file

**Comment by ArtyZe** [comment]
You can give me your personal emai ：）






At 2019-06-21 14:45:16, "abhigoku10" <notifications@github.com> wrote:


@zouwen198317 did u get the cfg and weight file
@ArtyZe can you share the cfg and weight file

**Comment by abhigoku10** [comment]
@ArtyZe abhigoku10@gmail.com is the mail id

**Comment by sicarioakki** [comment]
@sayanmutd  @aimhabo 
How have you guys used instance_segmenter.c with YOLO?
Basically, how is the instance_segmenter.c called with YOLO?

**Comment by abhigoku10** [comment]
@ArtyZe  thanks for sharing the model !!!!!!!

**Comment by sicarioakki** [comment]
@pjreddie Could you please elaborate as to how the module 'instance_segmenter.c' can be used with YOLO?
